mappings/net/minecraft/server/dedicated/DedicatedServer.mapping

```diff
@@ -1,7 +1,30 @@
 CLASS none/lf net/minecraft/server/dedicated/DedicatedServer
 	FIELD k LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD q serverProps Lnone/lc;
+	FIELD r eulaReader Lnone/kw;
+	FIELD s generateStructures Z
+	FIELD t gameMode Lnone/ais;
+	METHOD <init> (Ljava/io/File;Lnone/pf;Lcom/mojang/authlib/yggdrasil/YggdrasilAuthenticationService;Lcom/mojang/authlib/minecraft/MinecraftSessionService;Lcom/mojang/authlib/GameProfileRepository;Lnone/mn;)V
+		ARG 2 authService
+	METHOD B exit ()V
+	METHOD E isNetherAllowed ()Z
+	METHOD U isMonsterSpawningEnabled ()Z
 	METHOD Z isSnooperEnabled ()Z
+	METHOD a setGameMode (Lnone/ais;)V
+		ARG 0 gameMode
+	METHOD aD getMaximumWorldSize ()I
```

Comment:
Does this method affect the world border? If so, should it be `getWorldBorderRadius` instead?